cypress/e2e/web/technologyForm.js

```diff
@@ -278,7 +280,9 @@ describe('creating/editing technology', () => {
 
 			cy.get('@uploadedImages')
 				.children()
-				.should('have.length', 2);
+				.should(($div) => {
+					if ($div.length < 2) throw new Error();
+				});
```

Comment:
Acho que isso pode ser algo do tipo `should('have.length',2)` ?

Reply:
já era assim... porém, quando eu repetia o teste, durante as alterações, os uploads iam ficando no banco... e quando chegava nessa parte, não tinham mais 2 uploads, mas sim 4, 6, 8... tinha que ficar limpando o banco para rodar... assim ficou mais genérico... envio 2 imagens... tem que ter no mínimo 2 lá 😅

Reply:
Faz sentido! No caso isso aqui seria mais apropriado então. Seletor do chai

`expect('test').to.have.length.of.at.least(4)`

Reply:
Pronto... deixei assim:

```javascript
.should(($div) => {
    cy.expect($div.length).to.be.above(1);
});
```